README.md

```diff
@@ -119,7 +119,7 @@ reading it from the following sources, in this order:
 3. The first `.python-version` file found (if any) by searching each parent
    directory, until reaching the root of your filesystem.
 
-4. The global `~/.pyenv/version` file. You can modify this file using
+4. The global `.python-version` file. You can modify this file using
```

Comment:
`$(pyenv root)/version` directory doesn't exist.
Is the correct path `$(pyenv root)/versions`?